### ISSUE TITLE
Unsupported Browsers

### ISSUE THREAD

**Issue by BNNorman** [ConnectedHumber/Air-Quality-Web#34]
Just tried to access the AQ map using Internet Explorer - all I got was the heading - no map.

If a browser is not supported I think we should display a message to the user.

**Comment by sbrl** [comment]
Indeed. _Internet Explorer_ is not support by Microsoft (it's replacement is edge), and neither is it supported by me. I don't actually really have access to a Windows machine to test it on.

The primary browsers supported are _Firefox_ (I use Firefox for development - if it doesn't work in another browser try Firefox) and _Chrome_ (incl. derivatives). Safari should also work, but is untested (if not, open an issue).

Displaying such a message may be a bit of a challenge, but not impossible.

**Comment by sbrl** [comment]
Thinking about this, I'll probably need to see what the errors are in the _Internet Explorer Developer Tools_. I think you open them with either F11 or F12. Only when I know where it goes wrong will I be able to work out how to display a message to old browsers.

It could be it's failing on a syntax error because of new Javascript features, in which case I'll need to work around that using a different technique than I would otherwise.

Without access to a machine to test this on though, I can't be certain.

**Comment by BNNorman** [comment]
This is what I get with IE.

![image](https://user-images.githubusercontent.com/15849181/59994996-cc8dbd00-964d-11e9-8ab5-8734ec7bbfe6.png)

The map display works in Edge - never tried it before as I usually use Firefox.

**Comment by sbrl** [comment]
Ah, I see. It is a syntax error then as I suspected. Thanks!

**Comment by BNNorman** [comment]
Yup
